### PR TITLE
Migrated FeaturedGames() and CurrentGame to V3 -> fixing issue #351

### PR DESCRIPTION
**Pull request by Shidesu**
I tested this and should fix the issue #351 as mentionned in the title.
I'm waiting for your review @BenFradet 

(The empty lines were added by Resharper, I'm sorry)

### CODE REVIEW

**Comment by Shidesu** [comment]
I'm going to try to do this.

File: RiotSharp.sln.DotSettings.user [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/Environment/ContinuousTesting/PersistentDataLocation/@EntryValue">C:\Users\anchi\AppData\Local\Temp\JetBrains\ContinuousTesting\7c89bb27-c58f-4c7b-bc74-6120677e26e1</s:String>
+	<s:String x:Key="/Default/Environment/ContinuousTesting/SnapshotLocation/@EntryValue">C:\Users\anchi\AppData\Local\Temp\ssm.Wupufeg.tmp</s:String>
+	<s:String x:Key="/Default/Environment/ContinuousTesting/Strategy/@EntryValue">RunTests</s:String>
+	<s:String x:Key="/Default/Environment/ContinuousTesting/TriggerMode/@EntryValue">OnBuild</s:String></wpf:ResourceDictionary>
```
Comment (BenFradet) [inline RiotSharp.sln.DotSettings.user]: I don't think this was meant to be include?

File: RiotSharp/SpectatorEndpoint/CurrentGame/BannedChampion.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿using Newtonsoft.Json;
 
-namespace RiotSharp.CurrentGameEndpoint
+namespace RiotSharp.SpectatorEndpoint.CurrentGame
```
Comment (BenFradet) [inline RiotSharp/SpectatorEndpoint/CurrentGame/BannedChampion.cs]: Maybe I wasn't clear but I think you can flatten things and have only a `RiotSharp.SpectatorEndpoint` namespace and `SpectatorEndpoint` folder and have every class in this folder.

No need to have additional `CurrentGame` / `FeatureGames` folders and namespaces.

Sorry.

File: RiotSharp.sln.DotSettings.user (hunk 1 above)
Comment (Shidesu) [reply RiotSharp.sln.DotSettings.user]: Oh my bad. I will revert the commit and then change also for the SpectatorEndpoint

File: RiotSharp/SpectatorEndpoint/CurrentGame/BannedChampion.cs (hunk 2 above)
Comment (Shidesu) [reply RiotSharp/SpectatorEndpoint/CurrentGame/BannedChampion.cs]: I was unsure so I splitted in two but let's completely regroup.

**Comment by Shidesu** [comment]
This should now be okay 👍

**Comment by Shidesu** [comment]
Do it needs more modifications or it's fully okay ?

**Comment by BenFradet** [comment]
Looks good to me, waiting for a review from @JanOuborny

**Comment by JanOuborny** [comment]
@BenFradet I'm gonna be on it tomorrow

**Comment by Shidesu** [comment]
I was wondering -> should we cut the RiotApi in several little api ?

**Comment by BenFradet** [comment]
Indeed, it'd be cool to have multiple distinct endpoints and `RiotApi` would compose them all, e.g.:

```c#
var api = RiotApi.GetInstance();
api.Spectator.GetCurrentGame(region, id);
```

Feel free to log an issue!

**Comment by JanOuborny** [comment]
Seems reasonable, let's discuss the details of the implementation in an issue like @BenFradet already proposed. @Shidesu, are you on it or should I create the issue?

**Comment by Shidesu** [comment]
I think you should create the issue ^^

**Comment by Shidesu** [comment]
@BenFradet Should be okay here too, no ?

**Comment by BenFradet** [comment]
yup, waiting for @JanOuborny 's review

**Comment by BenFradet** [comment]
Meged thanks!

**Pull request merged**